docs/FAQ

```diff
@@ -520,7 +520,7 @@ FAQ
 
   3.3 Why doesn't my posting using -F work?
 
-  You can't simply use -F or -d at your choice. The web server that will
+  You can't simply use -F or -d as your choice. The web server that will
```

Comment:
Changed to "You can't arbitrarily use -F or -d, the choice between -F or -d depends on the 
  HTTP operation you need curl to do and what the web server that will receive 
  your post expects."